### PR TITLE
set SAFEPATH for Cygwin

### PR DESCRIPTION
**Pull request by Juppit**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -80,6 +80,9 @@ PLATFORM := $(shell uname -s)
 PATH := $(TOOLCHAIN)/bin:$(PATH)
 SAFEPATH := $(TOOLCHAIN)/bin:/usr/local/sbin:/usr/bin:/bin:/usr/sbin:/sbin:/usr/local/bin:/mingw/bin/:/c/tools/mingw32/bin:/c/tools/mingw64/bin
 
+ifneq (,$(findstring CYGWIN,$(PLATFORM)))
```
Comment (DimmKirr) [inline Makefile:83]: Let's add an `else` with

```
SAFEPATH := $(TOOLCHAIN)/bin:/usr/local/sbin:/usr/bin:/bin:/usr/sbin:/sbin:/usr/local/bin:/mingw/bin/:/c/tools/mingw32/bin:/c/tools/mingw64/bin
```

**Comment by DimmKirr** [comment]
Also, I think it would be better to try to keep all platform-specific stuff in `platform-specific` target, and pass it to makes there, if it's not just setting a variable

File: Makefile (hunk 1 above)
Comment (Juppit) [inline Makefile:83]: This is tested on cygwin, but I don't know what is right on ubuntu, other linux or BSD.
I'm pretty shure they don't need mingw in there path. ;-)
Or doyou mean.
if mingw then oldpath else myshortpath?

**Comment by Juppit** [comment]
atm I close this issue

**Pull request closed without merging**